e2e/pages/organizations/developers/developers.readonly.spec.js

```diff
@@ -28,6 +31,20 @@ describe('the Developer pages', () => {
         expect(page.products.length).toBeGreaterThan(0);
       });
     });
+
+    describe('when on the "Breeze EHR" Developer page with only one product', () => {
+      beforeEach(() => {
+        let developer = 'Breeze EHR';
+        page = new DevelopersPage();
+        page.selectDeveloper(developer);
+        page.getDeveloperPageTitle(developer).waitForDisplayed();
+        page.certificationStatusFilter(0);
```

Comment:
I see how this method works, but it would be helpful if it were a method with a useful name, maybe something like: 
```suggestion
        page.allAllCertificationStatuses();
```

Then the page could use the "set by index" method in the page object to do that, and if/when the implementation of how that filter works is changed, this test won't need to change, just the page's implementation of that action